conversant/demo/streamlit_example.py

```diff
@@ -28,15 +29,24 @@
 USER_AVATAR_SHORTCODE = ":bust_in_silhouette:"
 
 
+def peek(iterable):
```

Comment:
Can we have a brief one line docstring here?

Reply:
(also typehints :))